src/prefect/cli/concurrency_limit.py

```diff
@@ -25,7 +25,7 @@
     name="concurrency-limit",
     help="Commands for managing task-level concurrency limits.",
 )
-app.add_typer(concurrency_limit_app)
+app.add_typer(concurrency_limit_app, pluralization_string="concurrency_limits")
```

Comment:
Should this be `concurrency-limits`, or does `typer` make that change for us?  In either case, could we get an additional test that shows what happens with `snake_case` names (i.e. they are used as-is, they are converted to `kebab-case`, etc)

Reply:
whoops, this is a mistake on my part